block/pkg/datastore/driver/clean.go

```diff
@@ -0,0 +1,17 @@
+package driver
+
+type Closer interface {
```

Comment:
What is the usage of this interface and functions below?

Reply:
Removed